docs/general/emulator-support-and-issues.md

```diff
@@ -280,56 +310,49 @@ description: Comprehensive guide to emulator support and compatibility for Retro
 
 ### MS-DOS
 
-- ❌ _Not supported_ - needs hashing method and memory map
-- ❌ libretro core: **dosbox-core**
-- ❌ libretro core: **dosbox-SVN**
-- ❌ libretro core: **dosbox-pure**
-  - Needs a way to prevent launching with user-specified command line parameters
-  - Needs a way to prevent TSRs
-  - Needs a way to deactivate achievements if game drops to command prompt
+- ❌ _Not supported_ - needs a hashing method and a memory map.
+- ❌ libretro core: **DOSBox-core**
+- ❌ libretro core: **DOSBox-SVN**
+- ❌ libretro core: **DOSBox-Pure**
+  - Needs a way to prevent launching with user-specified command line parameters, TSRs and deactivate achievements if the game drops to the command prompt.
 
 ### MSX
 
 - ✅ libretro core: **blueMSX**
-  - Hash relies on .DSK file not being modified
-  - .DSK files appear to be unsupported if they also require a cartridge. There is a core option for this, but it doesn't seem to function properly. See "SD Snatcher", which used a 'sound' cartridge.
+  - The hash relies on the .dsk file not being modified.
+  - .dsk files appear to be unsupported if they also require a cartridge. There is a core option for this, but it doesn't seem to function properly. See "SD Snatcher", which used a 'sound' cartridge.
```

Comment:
Recommend removing this comment.  Not useful to players.

Reply:
Which one do you mean? The first one or the second one?

Reply:
Both of them.  I don't think anyone assumes you can modify game files and they still will hash properly.  The second comment doesn't actually help players either.